README.md

```diff
@@ -36,6 +37,10 @@ Spindexer works by:
    - IPFS pinning api url for pinning ipfs content to your own node
```

Comment:
maybe just change the comments here to something like:
   - IPFS gateway endpoint for accessing IPFS content
   - IPFS RPC endpoint for adding new content to IPFS
   - IPFS pinning api url for pinning ipfs content to a node

```

```